src/main/resources/codeOfConductPage.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Same as previous PR, you should create the current code of conduct json from our website. This file, for example you can copy for test.